addons/binding/org.openhab.binding.honeywellwifithermostat/.classpath

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry exported="true" kind="lib" path="lib/json-20141113.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/jsoup-1.8.2.jar"/>
```

Comment:
Same here: You are using this lib at a single place, mainly to find a certain string within an HTML page. I would assume that you can pretty easily work with StringUtils here and search for the substring you are looking for. Then this lib could be removed as well.